python/tk_desktop/desktop_engine_site_implementation.py

```diff
@@ -231,6 +249,79 @@ def run(self, splash=None, version=None):
         result = app.exec_()
         return result
 
+    def uses_legacy_authentication(self):
+        """
+        Returns if the Desktop's bootrap code uses the tk-framework-login for
+        logging in.
+
+        :returns: True the bootstrap logic is older than 1.1.0, False otherwise.
+        """
+        return StrictVersion(self.app_version) < StrictVersion("1.1.0")
```

Comment:
In other places of the code, we are using LooseVersion, just to cater for version numbers with RC suffixes, etc. Also, would it make sense to use the util methods here? https://github.com/shotgunsoftware/tk-core/blob/master/python/tank/deploy/util.py#L115